pkg/kvstore/chain_test.go

```diff
@@ -29,13 +28,12 @@ func TestChainKvStore_Contains(t *testing.T) {
 	element1.AssertExpectations(t)
 
 	// available
-	element0.On("Get", ctx, "foo", item).Return(false, nil).Once()
-	element1.On("Get", ctx, "foo", item).Run(func(args mock.Arguments) {
-		item := args[2].(*string)
-		*item = "bar"
+	element0.EXPECT().Get(ctx, "foo", item).Return(false, nil).Once()
+	element1.EXPECT().Get(ctx, "foo", item).Run(func(ctx context.Context, key any, value *string) {
+		*value = "bar"
 	}).Return(true, nil).Once()
 
-	element0.On("Put", ctx, "foo", "bar").Return(nil).Once()
+	element0.EXPECT().Put(ctx, "foo", "bar").Return(nil).Once()
```

Comment:
Do you already have golangci-lint v2 enabled? Passing a context.Background is discouraged afaik.
In any case I can recommend to use your shiny new matcher :D